ftplugin/elixir.vim

```diff
@@ -51,3 +51,19 @@ silent! setlocal formatoptions-=t formatoptions+=croqlj
 
 let b:undo_ftplugin = 'setlocal sw< sts< et< isk< com< cms< path< inex< sua< def<'.
       \ '| unlet! b:match_ignorecase b:match_words b:block_begin b:block_end'
+
+function s:projectionist_detect_elixir()
+  if filereadable('mix.exs')
+    let root = simplify(fnamemodify('.', ':p:s?[\/]$??'))
+
+    let projections = {}
+    let projections['lib/*.ex'] = { 'type': 'main', 'alternate': ['test/{}_test.exs'] }
```

Comment:
I prefer `type: source` to `type: main`, which follows the convention used by the Projectionist documentation. What do you think?

Reply:
I use `type: lib`